encrypted-media/scripts/playback-temporary.js

```diff
@@ -29,106 +25,37 @@ function runTest(config) {
                         event.messageType,
                         [ 'license-request', 'individualization-request' ] );
 
-            if ( event.messageType !== 'individualization-request' ) {
-                _events.push( event.messageType );
-            }
-
             config.messagehandler( config.keysystem, event.messageType, event.message ).then( function( response ) {
-                _events.push( 'license-response' );
 
-                waitForEventAndRunStep('keystatuseschange', _mediaKeySession, onKeyStatusesChange, test);
-                _mediaKeySession.update( response ).then( function() {
-                    _events.push('updated');
-                }).catch(function(error) {
+                _mediaKeySession.update( response ).catch(function(error) {
                     forceTestFailureFromPromise(test, error);
                 });
             });
         }
 
-        function onKeyStatusesChange(event) {
```

Comment:
I simplifies this basic test case to only test playback and not to fully verify the event sequence. There is a separate test which also verifies the event sequence. Otherwise it looks like the basic function of playback is broken when perhaps it is only a tiny problem with the event sequence.